jenkins/ci.suse.de/openstack-ardana-testbuild-gerrit.yaml

```diff
@@ -2,13 +2,6 @@
     name: openstack-ardana-testbuild-gerrit
     project-type: pipeline
     concurrent: true
-    wrappers:
-      - timestamps
```

Comment:
Why remove that?

Reply:
Ah, right, forgot to mention that wrappers no longer have any effect on Jenkins pipelines. That's why we use:

```
options {
    skipDefaultCheckout()
    timestamps()
}
```
in the Jenkinsfile.